lib/ngx/errlog.md

```diff
@@ -210,6 +211,30 @@ clear the table yourself before feeding it into the `errlog.get_logs` function.
 
 [Back to TOC](#table-of-contents)
 
+get_sys_filtering_level
+-----------------------
+**syntax:** *log_level = log_module.get_sys_filtering_level()*
+
+**context:** *any*
+
+Return system default filtering level as an integer. Later the return value
+could be used as Nginx log level constant. For example:
+
+```lua
+init_by_lua_block {
+    local errlog = require "ngx.errlog"
+    local log_level = error_log.get_sys_filtering_level()
+    -- Now the filter level is always one level lower than system default log level
```

Comment:
This is wrong I'm afraid. The smaller the log level values, the *higher* the priority of the level. See

```
#define NGX_LOG_STDERR            0
#define NGX_LOG_EMERG             1
#define NGX_LOG_ALERT             2
#define NGX_LOG_CRIT              3
#define NGX_LOG_ERR               4
#define NGX_LOG_WARN              5
#define NGX_LOG_NOTICE            6
#define NGX_LOG_INFO              7
#define NGX_LOG_DEBUG             8
```

So when you decrement the level, you actually make it one level *higher* instead of one level lower.